test/integration/Integration.t.sol

```diff
@@ -0,0 +1,640 @@
+        _unwrap(_bob, _bob, _bobBalance + _bobAccruedYield);
+
+        // Assert Bob (Earner)
+        assertEq(_mToken.balanceOf(_bob), _bobBalance + _bobAccruedYield);
+        assertEq(_wrappedMToken.balanceOf(_bob), _bobBalance -= _bobBalance);
+        assertEq(_wrappedMToken.accruedYieldOf(_bob), _bobAccruedYield -= _bobAccruedYield);
+
+        // Assert Globals
+        assertEq(_wrappedMToken.totalEarningSupply(), 100_000002);
+        assertEq(_wrappedMToken.totalNonEarningSupply(), 100_000000);
+        assertEq(_wrappedMToken.totalSupply(), 200_000002);
+        assertEq(_wrappedMToken.totalAccruedYield(), 2_496407);
+        assertEq(_wrappedMToken.excess(), _excess -= 2);
+
+        assertGe(
+            _wrapperBalanceOfM = _mToken.balanceOf(address(_wrappedMToken)),
+            _carolBalance + _carolAccruedYield + _daveBalance + _excess
+        );
+
+        // Accrued yield of Carol is claimed when unwrapping
+        _unwrap(_carol, _carol, _carolBalance + _carolAccruedYield);
+
+        // Assert Carol (Earner)
+        assertEq(_mToken.balanceOf(_carol), _carolBalance + _carolAccruedYield);
+        assertEq(_wrappedMToken.balanceOf(_carol), _carolBalance -= _carolBalance);
+        assertEq(_wrappedMToken.accruedYieldOf(_carol), _carolAccruedYield -= _carolAccruedYield);
+
+        // Assert Globals
+        assertEq(_wrappedMToken.totalEarningSupply(), 3);
+        assertEq(_wrappedMToken.totalNonEarningSupply(), 100_000000);
+        assertEq(_wrappedMToken.totalSupply(), 100_000003);
+        assertEq(_wrappedMToken.totalAccruedYield(), 4);
+        assertEq(_wrappedMToken.excess(), _excess -= 1);
+
+        assertGe(_wrapperBalanceOfM = _mToken.balanceOf(address(_wrappedMToken)), _daveBalance + _excess);
+
+        _unwrap(_dave, _dave, _daveBalance);
+
+        // Assert Dave (Non-Earner)
+        assertEq(_mToken.balanceOf(_dave), _daveBalance);
+        assertEq(_wrappedMToken.balanceOf(_dave), _daveBalance -= _daveBalance);
+        assertEq(_wrappedMToken.accruedYieldOf(_dave), 0);
+
+        // Assert Globals
+        assertEq(_wrappedMToken.totalEarningSupply(), 3);
+        assertEq(_wrappedMToken.totalNonEarningSupply(), 0);
+        assertEq(_wrappedMToken.totalSupply(), 3);
+        assertEq(_wrappedMToken.totalAccruedYield(), 4);
+        assertEq(_wrappedMToken.excess(), _excess -= 0);
+
+        assertGe(_wrapperBalanceOfM = _mToken.balanceOf(address(_wrappedMToken)), _excess);
+
+        uint256 vaultStartingBalance_ = _mToken.balanceOf(_vault);
+
+        assertEq(_wrappedMToken.claimExcess(), _excess);
+        assertEq(_mToken.balanceOf(_vault), _excess + vaultStartingBalance_);
+
+        // Assert Globals
+        assertEq(_wrappedMToken.totalEarningSupply(), 3);
+        assertEq(_wrappedMToken.totalNonEarningSupply(), 0);
+        assertEq(_wrappedMToken.totalSupply(), 3);
+        assertEq(_wrappedMToken.totalAccruedYield(), 4);
+        assertEq(_wrappedMToken.excess(), _excess -= _excess);
```

Comment:
```suggestion
        assertEq(_wrappedMToken.excess(), 0);
```

Reply:
I think the original l;ine is better at conveying that all the excess tracked is being decremented by the axcess removed.